src/apscheduler/datastores/sqlalchemy.py

```diff
@@ -389,6 +396,7 @@ async def start(
         async for attempt in self._retry():
             with attempt:
                 async with self._begin_transaction() as conn:
+                    await self._create_schema(conn)
```

Comment:
Is there a reason for this code to be in its own method? Is there a use case for making it overrideable by the user?

Reply:
You didn't answer my question yet.

Reply:
I just need to resolve this question, and then I'm willing to merge this PR. If indeed you want to make schema creation overrideable by users, then the condition should be here, and the method should just create the schema unconditionally. But I need to understand why you made it into a separate method here.

Reply:
I will add a param for this tmr.

Reply:
What parameter? I wasn't asking about any parameters. And don't resolve the conversation when my question remains unanswered.

Reply:
I think a separate method is easier to maintain.

Reply:
No, it's just more code. The only plausible reason for keeping it in a separate method is to allow for users to override the behavior, but I don't currently see any reason to let users do that. Additionally, that would increase the API footprint which I don't want to do unless there is a good reason.

Reply:
Can you modify it according to your suggestion？ thank you

Reply:
I've made the changes I wanted. I'll come back a while later to check on this and merge.